libraries/hatch-server-user-management/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@launchtray/hatch-server-user-management",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.js",
+  "license": "MIT",
+  "scripts": {
+    "prepare": "rimraf dist && tsc",
+    "build": "rimraf dist && tsc"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@types/cookie": "^0.4.0",
```

Comment:
Pretty sure a lot of these should be peer dependencies

Reply:
I updated but could you take a look and confirm

Reply:
After a deeper review, I think you were actually closer with your original list. Anything that is part of the public interface of the library should be a peer dependency. I think that includes only the following:

- hatch-server
- hatch-server-middleware
- hatch-util

I think the rest is implementation detail that is not exposed via the public API, but let me know if you disagree.

Reply:
Updated